build.gradle

```diff
@@ -229,6 +229,16 @@ dependencies {
     compile group: 'org.springframework.boot', name: 'spring-boot-starter-web', version: versions.springBoot
     compile group: 'org.springframework.cloud', name: 'spring-cloud-starter-netflix-hystrix', version: versions.springHystrix
 
+    compile('org.apache.logging.log4j:log4j-to-slf4j:2.13.2') {
```

Comment:
same question like in documents-api (https://github.com/hmcts/ia-case-documents-api/pull/254/files), do we want to force log4j or keep suppression? @raghera

Reply:
@makowalski As per the discussion I have removed the forced dependency for the above